mkdocs.yml

```diff
@@ -20,8 +20,20 @@ markdown_extensions:
 theme:
   name: "material"
   palette:
-    primary: "black"
-    accent: "orange"
+    - media: "(prefers-color-scheme: light)"
```

Comment:
Links have contrast issues
<img width="1741" alt="image" src="https://github.com/user-attachments/assets/d90509c1-139d-44e7-b769-25468277b0fd">

please override the dark theme, for example
```
[data-md-color-scheme="slate"] {
    --md-default-bg-color: hsla(var(--md-hue), 15%, 14%, 1);
}
```
it should give 5.14 contrast ratio for links